backend/src/backend/main.py

```diff
@@ -51,4 +53,28 @@ async def test(timestamp: int):
     return {"requested_on": requested_on, "received_on": received_on}
 
 
+@api.post("/books/add/", status_code=201)
+async def books_add(book: SchemaBook):
```

Comment:
I'd prefer the function's name to be action-like. `add_book()` for instance. We'll probably group the *routes* into dedicated modules sometime.

I don't think using `book` as variable name is appropriate as we should reserve that for our model instances. We can use something like `payload`, `book_payload`, or `book_data`. Not sure what's best yet so I'll let you choose one that distinctive from our DB Model.